rust/main/agents/scraper/src/db/message.rs

```diff
@@ -31,64 +32,54 @@ pub struct StorableMessage<'a> {
 }
 
 impl ScraperDb {
-    /// Get the dispatched message associated with a nonce.
+    /// Get the delivered message associated with a sequence.
     #[instrument(skip(self))]
-    pub async fn retrieve_message_by_nonce(
+    pub async fn retrieve_delivered_message_by_sequence(
```

Comment:
nit: wonder if it's worth having the name reflect that this returns the message ID (not the message itself)

Reply:
Renamed